squad/ci/tasks.py

```diff
@@ -35,3 +38,33 @@ def submit(self, job_id):
         test_job.save()
         if issue.retry:
             raise self.retry(exc=issue, countdown=3600)  # retry in 1 hour
+
+
+@celery.task(bind=True)
+def send_admin_email(self, job_id):
```

Comment:
since you don't use `self` in the body of the function, I would drop the use of `@celery.bind` (and therefore the `self` argument)